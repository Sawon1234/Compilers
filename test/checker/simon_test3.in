class Branch(address : String, name : String, value : Integer, managerName : String) extends Integer
{
    super();
    fun getAddress() : String = address;
    fun getName() : String = name;
    fun getValue() : Integer = value;
    fun getManagerName() : String = managerName;
}

return ["sup"];