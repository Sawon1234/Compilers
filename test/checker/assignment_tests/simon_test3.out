class Nested < > ( ) extends Thing { return 1 ; return 2 ; return 3 ; super ( ) ; } return 4 ;